piezo_web_app/PiezoWebApp/tests/utils/dict_argument_helper.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+import pytest
+
+from PiezoWebApp.src.utils.dict_argument_helper import set_value_in_nested_dict
+
+
+class TestDictArgumentHelper(unittest.TestCase):
+
+    def test_set_value_in_nested_dict_sets_value_at_top_level_when_path_has_len_one(self):
+        # Arrange
+        path = ["x"]
+        nested_dict = {"a": 1, "b": {"c": 2}, "d": {"e": {"f": 3}}}
+        value = 5
+        # Act
+        result = set_value_in_nested_dict(nested_dict, path, value)
+        # Assert
+        assert result == {"a": 1, "b": {"c": 2}, "d": {"e": {"f": 3}}, "x": 5}
+
+    def test_set_value_in_nested_dict_sets_nested_values_for_paths_longer_than_one(self):
+        # Arrange
+        path = ["d", "e", "g"]
+        nested_dict = {"a": 1, "b": {"c": 2}, "d": {"e": {"f": 3}}}
+        value = 5
+        # Act
+        result = set_value_in_nested_dict(nested_dict, path, value)
+        # Assert
+        assert result == {"a": 1, "b": {"c": 2}, "d": {"e": {"f": 3, "g": 5}}}
```

Comment:
Could you please add a test where one of the original nested dictionaries has multiple key-value pairs? E.g.
```
nested_dict = {"a": 1, "b": {"c": 2}, "d": {"e": {"f": 3, "g": 4}}}
```

Also, we only ever seem to be adding new values in these tests - what happens when we overwrite old ones?